### ISSUE TITLE
COR.lua - Luzaf issues - Attempt to concatenate a boolean value

### ISSUE THREAD

**Issue by ValafarXI** [Kinematics/GearSwap-Jobs#71]
Hey folks, 

I've been playing around with COR .lua's and Kinematics' has been a great one to reference, but it's kicking up an odd error for me when I use it:

> Gearswap has detected an error in the user function self_command:
> ...)Windower4/addons/GearSwap/data/Kinematics_Cor.lua:342: attempt to concatenate a boolean value

In the script, line 342 reads:

> msg = msg .. ', Roll Size: ' .. (state.LuzafRing.value and 'Large') or 'Small'

After some time looking through it, I'm unable to determine the problem or what a possible fix might be - any suggestions?
Interestingly, this error _only_ occurs when the Luzaf ring variable (gs c toggle LuzafRing) is set to "off". While active, the script appears to run just fine.

Any help or fixes are greatly appreciated!

**Comment by Kinematics** [comment]
It's missing a set of parentheses.  Should be:

```
    msg = msg .. ', Roll Size: ' .. ((state.LuzafRing.value and 'Large') or 'Small')
```

Will push a fix.

**Comment by Kinematics** [comment]
Fixed and merged in 2610b0cca68aab6a32056432f12c94e82a9f5462
